numpy/add_newdocs.py

```diff
@@ -5463,18 +5442,22 @@ def luf(lamdaexpr, *args, **kwargs):
 
     Parameters
     ----------
-    X : array_like
-        First input array.
-    Y : array_like
-        Second input array.
-    out : array_like
-        An array to store the output. Must be the same shape as the
-        output would have.
+    *x : array_like
```

Comment:
This file never actually reaches sphinx anyway, but I was under the impression that this worked correctly due to the transformation that the `Parameters` section undergoes